apps/web/src/app/(dashboard)/documents/delete-document-dialog.tsx

```diff
@@ -81,7 +92,7 @@ export const DeleteDocumentDialog = ({
         </DialogHeader>
 
         {status !== DocumentStatus.DRAFT && (
-          <div className="mt-8">
+          <div className="mt-4">
```

Comment:
After testing various top margins, it seems that this works well without any margin.

What do you think?

![CleanShot 2023-12-20 at 17 41 37](https://github.com/documenso/documenso/assets/25515812/02d963fe-9e23-4165-819f-d16bd33abb87)

Reply:
I also felt the same, one of the previous PRs introduced the `mt-8` and I felt the margin was too huge as well

Reply:
That would be me, I like having the action buttons separated from any kind of input/form elements to assist with misclicking

Reply:
In the example above, it doesn't seem like it would be easy to misclick. Or am I missing something? 👀